.github/workflows/build.yml

```diff
@@ -25,8 +25,10 @@ jobs:
         with:
           go-version: '1.21'
 
-      - name: Build
```

Comment:
Does the indentation change mess it up? :grimacing:

Reply:
what indentation change? I'm not seeing one now.